pages/vi/vi-github-issues.md

```diff
@@ -124,6 +124,29 @@ Note: Remember to always sync your fork before starting to work on a new issue.
 
 This is an exercise to help you familiarize with GitHub issues, committing, and creating pull requests. This is a common process in large open source projects as there is always room for improvement. So, we strongly encourage you to follow this process and continue to post issues and resolve them.
 
+## Summary of steps to resolving an issue and generating pull request
+
+### Creating a branch 
+1.  Open an issue within the right repository (upstream repository) and explain the problem.
+If the issue is approved, `cd` to `yourusername.github.io` directory and sync with the upstream repository using `git fetch upstream` , ``git checkout master``, and ``git merge upstream/master``.
+
+2.  Use `git checkout -b name_of_your_new_branch` to create and switch to a new branch on your local repository. Check which branch you are on using `git checkout branch-name` before you make the desired changes to local files.
```

Comment:
Seems a bit redundant to use almost the same commands twice. If the objective is just to check, Shouldn't we use `git branch` for that ?